app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -66,7 +66,7 @@ function DocumentsStep({
         />
       ))}
       {isLastStep ? <FormStepsButton.Submit /> : <FormStepsButton.Continue />}
-      <DocumentCaptureTroubleshootingOptions hasErrors={!!errors.length} />
+      <DocumentCaptureTroubleshootingOptions />
```

Comment:
> I wonder if `hasErrors` should be renamed to something more specific to IPP `hasDocCaptureErrors` or something

I was on the fence about making some further revisions to the props, because I also think there's some redundancy between `showInPersonOption` and `hasErrors`, at least in how they're both currently used exclusively in determining whether to show in-person troubleshooting options.

https://github.com/18F/identity-idp/blob/c2a3697fbab4a5ed5c0200406f6c2f47995f8834/app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.tsx#L26-L34

For example, we could collapse them to a single prop, and maybe name it in such a way that allows for future off-ramp alternative proofing options to use (e.g. `showAlternativeProofingOptions`), especially since we're also still considering `inPersonURL` to control whether the feature is available for the current service provider.

Reply:
> > I wonder if `hasErrors` should be renamed to something more specific to IPP `hasDocCaptureErrors` or something
> 
> I was on the fence about making some further revisions to the props [...]

I gave this a shot in dfad07154.